packages/sdk/src/subscribe/Subscription.ts

```diff
@@ -37,13 +39,15 @@ export class Subscription extends MessageStream {
         streamPartId: StreamPartID,
         isRaw: boolean,
         erc1271ContractAddress: EthereumAddress | undefined,
+        deliveryOptions: StreamPartDeliveryOptions,
```

Comment:
As this is an optional configuration:

```suggestion
        deliveryOptions: StreamPartDeliveryOptions | undefined,
```